src/components/chip-set/chip-set.tsx

```diff
@@ -418,21 +416,30 @@ export class ChipSet {
                     <div class="mdc-notched-outline__trailing" />
                 </div>
                 {this.renderLeadingIcon()}
+                {this.renderEmptyValueForReadonly()}
                 {this.renderClearAllChipsButton()}
             </div>,
             this.renderHelperLine(),
         ];
     }
 
+    private renderEmptyValueForReadonly = () => {
+        if (this.readonly && this.value.length === 0) {
+            return (
+                <span class="lime-empty-value-for-readonly lime-looks-like-input-value">
+                    –
+                </span>
+            );
+        }
+    };
+
     private renderLabel() {
         const labelClassList = {
             'mdc-floating-label': true,
             'mdc-text-field--no-label': !this.label,
-            'mdc-text-field--disabled': this.readonly || this.disabled,
```

Comment:
I'm not sure if it is necessary, I couldn't notice any difference when removed  it 

https://user-images.githubusercontent.com/50618208/234785062-3bcc0d8e-2a5b-47c2-9c83-11785d744071.mov

Reply:
In other components, this has some function. Now I haven't studied this component again. But if you want to remove this class, you must do it in another commit and explain why it was a bad idea to have the class. 

Otherwise, you shouldn't remove this class in this commit. Because:
1. this will force the reviewer to think that you removed it to fix the problem. But as you say, it has nothing to do with fixing the problem.
2. if we have to revert this commit in a future version because we discover another bug, then we will have to a lot of detective work again to see if it's because of the class that was removed here (part of this commit), or if it is because of the rest of the things that happened here.

Moreover, it's better to be consistent across all the components. We have so many issues already because the structure and logic of this component is a copy of `limel-input-field`. In some cases they differ, and there is a reason for it. But we should keep the differences to a minimum.

Reply:
So now I'm checking to see why we have this class on the `mdc-floating-label`... it seems strange. The main component is getting the class already. So why repeat it inside the component on an element. This seems wrong.

Reply:
I think this whole line is unnecessary after testing. We can remove the whole line.
But this should be another refactor commit. Should I fix this, or do you want to do it?

Reply:
I will make a separate commit

Reply:
> We can remove the whole line.

I don’t think we should remove the whole line.  We still need this class for the `disabled` state

Reply:
it's on the floating label. We already have the same class on the main container.
So this must have been a mistake from the beginning

Reply:
but when I removed this line chipsets are not getting a disabled effect: 
 

https://user-images.githubusercontent.com/50618208/234803560-86feb5b6-18bf-4bdb-bca3-b732225a8d55.mov

Reply:
I think we can remove only the `readonly` condition but keep `disabled`

Reply:
you're on the wrong line Lucy 😅 
we're talking about line 439 in `private renderLabel() `

Reply:
true! 😁